tests/acceptanceTests/configs/sociable_config.txt

```diff
@@ -0,0 +1,5 @@
+This configuration file is used by the for acceptance test: AcceptanceTests_sociable.js.
```

Comment:
"This configuration file is used by the for ..." -> "The configuration file sociable_config.json is used by the ... "